### DISCUSSION TITLE
ODTT tool issue

### DISCUSSION THREAD

**Discussion by DHecosystem** [onvif/testspecs#66] [Q&A] [answered]
Dear ONVIF Team,

I received some feedback from our R&D on using ODTT tool to pass the test and add products on the portal. Below I list the 4 main points:
1. The tool failed to obtain PTZ capability - It often fails to obtain the PTZ capability of the device, prompting with error message that the device does not return PTZ information. It results in multiple times of obtaining such information. Please see screenshot_1 in attachment,
2. Devices with large resolution cannot pass the test – The same series device with lower resolution however can pass the test. Does the tool have any restriction on device resolution? Please see screenshot_2 as a reference.
3. The version 23.06 tool gets stuck occasionally during execution - The tool does not crash, but the number of use case execution does not change, and the test progress does not change for more than 4-5 hours.
4. Execution pass rate is low – Even the network environment and the test environment remain unchanged, the test results of each test case might be different. The same test case that succeeded last time may fail next time.

Please let me know if need more information.
Thank you.
![screenshot_1](https://github.com/onvif/testspecs/assets/155931688/f6ba65ba-b563-43a0-b1df-cc2fcb528f48)
![screenshot_2](https://github.com/onvif/testspecs/assets/155931688/78217fca-60f9-40e2-9128-a887a0a5d86c)

**Comment by madhurao68** [comment, marked as answer]
Hi,
Could you provide us with some additional information:
  - " Devices with large resolution cannot pass the test" - What is the resolution?
  -  What is the configuration of the test machine?  RAM, processor?

Also, for the first issue we would need logs (Wireshark or xml) from the tool.
  
Thanks!
